dom/rx-typings.d.ts

```diff
@@ -1,7 +0,0 @@
-import {EventsFnOptions} from './lib/DOMSource';
-import {Observable} from 'rx';
-export interface DOMSource {
-  select(selector: string): DOMSource;
-  elements(): Observable<Element>;
-  events(eventType: string, options?: EventsFnOptions): Observable<Event>;
-}
```

Comment:
Why remove this? I'm quite sure we still need it, as TS users will `import {DOMSource} from '@cycle/dom/rx-typings'` and also for rxjs, xstream, most.